timescaledb/how-to-guides/continuous-aggregates/adding-automatic-refresh-policies.md

```diff
@@ -94,3 +94,4 @@ DROP MATERIALIZED VIEW conditions_summary_hourly;
 
 [sec-data-retention]: /hot-to-guides/data-retention
 [api-drop-chunks]: /api/:currentVersion:/hypertable/drop_chunks
+[api-add-continuous-aggregate-policy]: /api/:currentVersion:/continuous-aggregates/add_continuous_aggregate_policy
```

Comment:
```suggestion
[api-add-continuous-aggregate-policy]: api/add_continuous_aggregate_policy/
```

Reply:
@noctarius is correct here. Currently the "top level" sections (TimescaleDB, API, Forge, Cloud) are their own section. If I want to reference one from another, I need to start higher on the URL and reference whatever version is currently selected in the dropdown. It might be "latest", but if we want to be ready for older versions, the `:currentVersion:` marker is replaced at build time with the correct link.